Eh aws marketplace (#2044)
huggingface/blog: enterprise_hub_aws_marketplace.md

Reviewer: use dashes instead of underscores for consistency

Author: done now it is titled "enterprise-hub-aws-marketplace"

---

> title: "The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account"

Reviewer 2: ```suggestion
title: "Subscribe to Enterprise Hub with your AWS Account"
```

thumbnail: /blog/assets/158_aws_marketplace/thumbnail.jpg
authors:
- user: Violette
- user: sbrandeis
- user: jeffboudier
---

> # The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account.

Reviewer 2: ```suggestion
# Subscribe to Enterprise Hub with your AWS Account
```

> AWS Enterprise users can now use Hugging Face as a safe & private collaboration platform with all billing handled directly on AWS.

Reviewer 2: ```suggestion
You can now upgrade your Hugging Face Organization to Enterprise using your AWS account - get started [on the AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-n6vsyhdjkfng2).
```

> AWS Enterprises users can now use Hugging Face as a safe & private collaboration platform without sharing payment information with Hugging Face.

Reviewer: ```suggestion
AWS Enterprise users can now use Hugging Face as a safe & private collaboration platform with all billing handled directly on AWS.
```

> ## What is the Enterprise Hub, the famous 🤗 Hub but in Enterprise mode? 

Reviewer 2: ```suggestion
## What is Enterprise Hub? 
```

> In addition to offering widely adopted open-source libraries and thousands of models and datasets, we enable an enterprise-ready version of the world’s leading AI platform. With the Enterprise Hub, you give your team the most advanced GDPR & SOC2T2 compliant platform to build AI with open-source resources complying with privacy and security Enterprise constraints: 

Reviewer 2: ```suggestion
Enterprise Hub is a premium subscription to upgrade a free Hugging Face organization with advanced security features, access controls, collaboration tools and compute options. With Enterprise Hub, companies can build AI privately and securely within our GDPR compliant and SOC2 Type 2 certified platform. Exclusive features include: 
```

> - Benefit from state-of-the-art machine learning, 
> - Run inference workloads, 
> - Compute budget for your team, 
> - Choose the location of your Hugging Face repos/storage, 
> - Use your company's SSO with hf.co and increase security & productivity
> - Enable teams in regulated environments to keep up with the pace of open-source advancement frictionlessly

Reviewer 2: ```suggestion
- Single Sign-On: ensure all members of your organization are employees of your company.
- Resource Groups: manage teams and projects with granular access controls for repositories.
- Storage Regions: store company repositories in Europe for GDPR compliance.
- Audit Logs: access detailed logs of changes to your organization and repositories.
- Advanced Compute Options: give your team higher quota and access to more powerful GPUs.
- Private Datasets Viewer: enable the Dataset Viewer on your private datasets for easier collaboration.
- Train on DGX Cloud: train LLMs without code on NVIDIA H100 GPUs managed by NVIDIA DGX Cloud.
- Premium Support: get the most out of Enterprise Hub and control your costs with dedicated support.
```

> How can I enable the Enterprise Hub with my AWS account?

Reviewer 2: ```suggestion
If you're admin of your organization, you can upgrade it easily with a credit card. But how do you upgrade your organization to Enterprise Hub using your AWS account? We'll walk you through it step by step below.
```

To confirm it, you can open the Billing settings for [your organization account](https://huggingface.co/settings/organizations), where you should now see a `subscribe-success` status.

![Subscription Confirmed](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/06_success.jpg "Subscription Confirmed")

### 3. Activate the Enterprise Hub for your team and unlock new features

If you want to enable the Enterprise Hub and use your organization as a private and safe collaborative platform for your team to build AI with open source, please follow the steps below.

Open the Billing settings for your organization, click on the ‘Enterprise Hub’ Tab, and click on “Subscribe Now”

![Subscribe Now](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/08_subscribe.jpg "Subscribe Now")

Now select the number of Enterprise Hub seats you are willing to buy for your organization, the billing frequency and click on Checkout. 

![Select Seats](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/09_select.jpg "Select Seats")

### Congratulations! 🥳

> Your organization is now subscribed to the Enterprise Hub, and billing is directly managed by your AWS account. All members of your Enterprise Hub can now start using safely and privately the Hugging Face Hub.

Reviewer 2: ```suggestion
Your organization is now upgraded to Enterprise Hub, and its billing is directly managed by your AWS account. All members of your organization can now benefit from the advanced features of Enterprise Hub to build AI privately and securely.
```

---
title: "The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account"
thumbnail: /blog/assets/158_aws_marketplace/thumbnail.jpg
authors:
- user: Violette

> - user: philschmid

Reviewer 3: ```suggestion
```
